src/components/index.d.ts

```diff
@@ -1030,7 +1029,6 @@ export declare const NoraDrawer: {
     labelCopy: string
     closeCopy: string
     position?: string
-    className?: string
```

Comment:
NoraDrawer does not have this prop. 

https://github.com/getethos/ethos-design-system/blob/0fb56f9e2680be162c6c11b8b4372d28126c4cfe/src/nora/components/NoraDrawer/NoraDrawer.js#L44-L50